kong/init.lua

```diff
@@ -347,57 +522,80 @@ function Kong.balancer()
     end
   end
 
-  current_try.ip   = addr.ip
-  current_try.port = addr.port
+  local ip, port = addr.ip, addr.port
```

Comment:
style: we should avoid multiple declarations per line, maybe something we should add to our style guide